main.js

```diff
@@ -0,0 +1,98 @@
+///////////////////////////////////////Variables
+
+var i = 0; i < 10;
+// var connect3 = (["", "", ""] || ["", "", ""] || ["", "", ""]);
+
+///////////////////////////////////////Game Controller
+
+function turnAlternator(){
+  changeTurn[i] = document.addEventListener('click', function(event){
+    i++
+
+  })
+}
+
+// $(function({
+//   $('td').on('click', function(){
+//     i++
+//   })
+//   })
+// })
+
+function changeTurn(){
+
+    if (i % 2 === 0){
+      $('td').on('click', function(){
+      $(this).addClass('X').html("X")
+
+      console.log('ivebeen clicked by x');
+    })
+    } else {
+      $('td').on('click', function(){
+      $(this).addClass('O').html("O")
+
+      console.log('ive been clicked by o');
+    })
+  }
+}
+
+
+
+///////////////////////////////////////Clickable reset
+
+$(function(){
```

Comment:
This works fine, but I would add an event listener to the reset button and have it call a specific named function called "resetBoard" or something that you define to do this.